rholang/src/test/scala/coop/rchain/rholang/interpreter/ReduceSpec.scala

```diff
@@ -115,7 +115,8 @@ class ReduceSpec extends FlatSpec with Matchers with PersistentStoreTester {
                 channels,
                 bindPatterns,
                 TaggedContinuation(ParBody(body)),
-                false
+                false,
```

Comment:
Judging from the commit name the previous one does not compile or it's tests are failing. These should be squashed then

Reply:
I don't mind. My focus was to have small commits that introduce concepts step by step. changing a lot of unrelated files seemed like purposefully making the previous commit less obvious.